packages/nextjs/pages/builders/0x2F142f9F52af0845B7b8b1353E349Ca1Cb8EFE92.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { createPublicClient, http } from "viem";
+import { mainnet } from "viem/chains";
+
+export function PapaWheelie() {
+  const [ensData, setEnsData] = useState<Record<string, string | undefined>>({});
+  const nameTitle = "P A P A W H E E L I E";
+  const ensName = "papawheelie.eth";
+
+  useEffect(() => {
+    const client = createPublicClient({
```

Comment:
Instead of instantiating a new mainnet client, you can use the one declared on wagmiConnectors (we always have a mainnet client so we can access ENS data :))

Check: https://wagmi.sh/react/api/hooks/useClient

Reply:
Great! You could also use `mainnet` from `viem/chains` if you don't want to hardcode the value.